templates/repo/home.tmpl

```diff
@@ -137,6 +137,20 @@
 				{{end}}
 			</div>
 		</div>
+		<!-- Only show clone panel in repository home page -->
+		{{if and (eq $n 0) .haveCitationFile}}
+			<div class="ui stackable secondary menu mobile--margin-between-items mobile--no-negative-margins no-vertical-tabs">
+				<div class="fitted item">
+					Cite this repository :&nbsp;
+					<div class="ui action tiny input" id="citation-panel">
+						{{template "repo/citation_buttons" .}}
+						<button id="goto-citation-btn" class="ui basic jump icon button tooltip" onclick="window.location='{{$.RepoLink}}/src/{{$.BranchName}}/CITATION.cff';" data-position="top right">
```

Comment:
Please also try to avoid using inline scripts here and instead use something like `<a href="...">`